src/models/index.tsx

```diff
@@ -26,5 +22,6 @@ export interface StarRatingProps {
   numberOfStars: number;
   colorFilled: string;
   colorUnfilled: string;
+  voteAverage: number;
```

Comment:
any reason this is interface not a type on component file?

Reply:
![image](https://user-images.githubusercontent.com/41084902/115006044-16920380-9eb1-11eb-9921-b28d395e70ab.png)

Reply:
why not 
type StarRatingProps = {
  numberOfStars: number;
  colorFilled: string;
  colorUnfilled: string;
  voteAverage: number;
}
on StarRating component?